### PR TITLE
EVA-429: Google analytics 

### PR DESCRIPTION
**Pull request by kandaj**
EVA-429

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -220,12 +220,12 @@
                 <div>
                     <h3>EVA Submissions</h3>
                     <p>Unless fully described in the header of the VCF file(s), EVA submissions are accompanied by an Excel spreadsheet and/or tab-delimited text files which describe the meta-data. We encourage our users to submit as much metadata as possible.</p>
-                    <p>Please download our submission template via this <a href="files/EVA_Submission_template.EVA_.V1.0.1_live.xlsx" target="_blank">link</a>.</p>
-                    <p>A completed template for a fictional submission can be downloaded <a href="files/EVA_Submission_template.EVA_.V1.0.1_live_mockup.xlsx" target="_blank">here</a>.</p>
+                    <p>Please download our submission template via this <a href="files/EVA_Submission_template.EVA_.V1.0.1_live.xlsx" target="_blank"  onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'EVA_Submission_template.EVA_.V1.0.1_live.xlsx'})">link</a>.</p>
```
Comment (cyenyxe) [inline src/index.html]: Shouldn't this file (and as a result the link) be the same as in line 193?

File: src/index.html [hunk 2]
```diff
@@ -856,6 +857,25 @@ <h3 class="about"><a href="//www.ebi.ac.uk/about">About us</a></h3>
...
 
+        function loadGoogleAnalytics() {
+            var trackingID;
+            if (document.location.hostname == 'www.ebi.ac.uk') {
+                trackingID = 'UA-82688737-1';
+            } else if (document.location.hostname == 'wwwdev.ebi.ac.uk') {
+                trackingID = 'UA-82375858-1';
+            } else {
+                trackingID = 'UA-82386453-1';
+            }
```
Comment (cyenyxe) [inline src/index.html]: Could the presence of these IDs in the code cause any security vulnerabilities?

Comment (kandaj) [inline src/index.html]: this shouldn't be a problem we can add a filter in the google analytics. 
ex: https://blog.kissmetrics.com/protect-analytics-from-hacking/

Comment (cyenyxe) [inline src/index.html]: Please do, taking into account this comment: https://blog.kissmetrics.com/protect-analytics-from-hacking/#comment-27835

File: src/js/clinvar/eva-clinical-widget-panel.js [hunk 3]
```diff
@@ -540,6 +540,9 @@ EvaClinicalWidgetPanel.prototype = {
 
         var newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + '?' + 'Clinical Browser&' + $.param(values);
         history.pushState ('forward', '', newurl);
+        //sending tracking data to Google Analytics
+        ga('send', 'event', { eventCategory: 'Search', eventAction: 'Clinical Browser', eventLabel: $.param(values)});
```
Comment (cyenyxe) [inline src/js/clinvar/eva-clinical-widget-panel.js]: Is there any criteria (described in a tutorial or something) to assign 'eventCategory' and 'eventAction'? To me, it looks like these 2 should be swapped: an action "search" is performed in a category "Clinical Browser", not the opposite.

File: src/index.html [hunk 4]
```diff
@@ -164,7 +164,7 @@
                 <div>
                     <p>
                         <!--<img src="img/Receive.png" alt="" width="48" height="48"> &nbsp;Receive your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:-->
-                        Download your <a href="files/eva_submission_package.zip" target="_blank">submission pack</a>, which will include:
+                        <!--Download your <a href="files/eva_submission_package.zip" target="_blank" onClick="ga('send', 'event', { eventCategory: 'Download', eventAction: 'Click', eventLabel: 'eva_submission_package.zip'});">submission pack</a>, which will include:-->
```
Comment (cyenyxe) [inline src/index.html]: Could the submission template changes be extracted into another PR? They don't relate to Google Analytics, but affect the submission process quite strongly.

Comment (kandaj) [inline src/index.html]: Another PR in the sense undo changes here and do it another branch?

Comment (cyenyxe) [inline src/index.html]: It seems the last commit was the one that introduced them. So it would mean taking it out from the history, undoing those specific changes and committing only the GA related ones in this branch. Then yes, the submission template changes would be placed in a new branch.

Comment (kandaj) [inline src/index.html]: Done.  Created another PR for submission template update.
